client/src/api/block_builder/input_selection/automatic.rs

```diff
@@ -170,6 +171,7 @@ impl<'a> ClientBlockBuilder<'a> {
                     let mut input_selection = InputSelection::new(
                         available_inputs.clone(),
                         self.outputs.clone(),
+                        Vec::new(),
```

Comment:
Wrong :trollface:

Reply:
Yeah I'll create an issue to bring it back, I won't have time to fix it and it's not important until we release a client crate version

Reply:
https://github.com/iotaledger/iota.rs/issues/1512